packages/jest-mock-apollo/README.md

```diff
@@ -10,16 +10,24 @@ $ yarn add @shopify/jest-mock-apollo
 
 ## Setup
 
-This package provides a method, `registerSchema`, which should be called in the jest `setup` file. For example:
+This package provides a factory class, which should be instantiated in a `utilities` file for your project. For example:
```

Comment:
I find "in a `utilities` file" a bit over prescriptive and also a little vaguely named